pino-elasticsearch.js

```diff
@@ -65,16 +68,10 @@ function pinoElasticSearch (opts) {
       })
     },
     write: function (body, enc, cb) {
-      const obj = {
-        index,
-        type,
-        consistency,
-        body
-      }
+      const obj = {index, type, consistency, body}
       client.create(obj, function (err, data) {
         if (!err) {
-          data.body = body
-          splitter.emit('insert', data)
+          splitter.emit('insert', {data: data, body: body})
```

Comment:
why this breaking change?  can't we just do `splitter.emit('insert', data, body)`?